api/circulation.py

```diff
@@ -200,9 +216,14 @@ def borrow(self, patron, pin, licensepool, delivery_mechanism,
         internal_format = api.internal_format(delivery_mechanism)
 
         if patron.fines:
-            max_fines = Configuration.max_outstanding_fines()
-            if patron.fines >= max_fines.amount:
-                raise OutstandingFines()
+            def parse_fines(fines):
```

Comment:
I'm pretty sure this is a reversion of code I committed earlier. Calling `Configuration.max_outstanding_fines()` is the correct behavior.